conf/messages.cy

```diff
@@ -462,6 +462,9 @@ soleTraderEntityVerification.failure.detail  = Gwiriwch eich cofnodion Hunanases
 
 duplicateSubscription.title  = Mae’ch sefydliad eisoes wedi’i gofrestru
 duplicateSubscription.detail  = Mae rhywun yn {0} eisoes wedi cofrestru ar gyfer y Dreth Deunydd Pacio Plastig.
+duplicateSubscription.account.detail.link = [Ewch i’ch cyfrif Treth Deunydd Pacio Plastig i wneud y canlynol]:
```

Comment:
Hi Carl, I am not sure if you have seen my previous comments, but I asked Lisa about the brackets and she said to leave them in. But fell free to ask her agin as she might misunderstood me.